.azure/wso2-opensearch-k8s-operator-release.yaml

```diff
@@ -0,0 +1,61 @@
+resources:
+  repositories:
+  - repository: common-templates
+    type: github
+    name: wso2-enterprise/choreo-common-pipeline-templates
+    endpoint: wso2-enterprise
+
+trigger:
+  batch: true
+  branches:
+    include:
+    - wso2-main
+
+pr: none
+
+variables:
+- name: CONTAINER_REGISTRY
+  value: choreocontrolplane.azurecr.io
+- name: REPOSITORY
+  value: choreoipaas/wso2-opensearch-k8s-operator
+- name: APP_REVISION
+  value: $(Build.SourceBranchName)-$(Build.SourceVersion)
+
+pool:
+  vmImage: "ubuntu-latest"
+
+steps:
+- task: Docker@2
+  displayName: "Docker Build"
+  inputs:
+    command: build
+    Dockerfile: "opensearch-operator/Dockerfile"
+    buildContext: "opensearch-operator"
+    containerRegistry: "wso2choreo-control-plane-acr"
+    repository: $(REPOSITORY)
+    tags: $(APP_REVISION)
+- template: install/install-trivy.yaml@common-templates
+- template: pr/trivy-scan.yaml@common-templates
+  parameters:
+    IMAGES:
+    - $(CONTAINER_REGISTRY)/$(REPOSITORY):$(APP_REVISION)
+- script: |
+    docker tag $(CONTAINER_REGISTRY)/$(REPOSITORY):$(APP_REVISION) $(PRIVATE_CONTAINER_REGISTRY)/$(REPOSITORY):$(APP_REVISION)
+    docker tag $(CONTAINER_REGISTRY)/$(REPOSITORY):latest $(PRIVATE_CONTAINER_REGISTRY)/$(REPOSITORY):latest
```

Comment:
there is no docker build for latest tag it seems

Reply:
Fixed through https://github.com/wso2/wso2-opensearch-k8s-operator/pull/2/commits/c901a4aceaa9c4fa05f3ed9e0217b632301ddfb5